static/usage/v8/theming/class-dark-mode/theme/variables_css.md

```diff
@@ -0,0 +1,33 @@
+```css
+/* Ionic Variables and Theming
+ * ---------------------------------------------------------------
+ * Any overrides to theme variables should be placed in this file.
+ * For more information, please see:
+ * http://ionicframework.com/docs/theming/
+ */
+
+/* This sets a different item border color for the default theme on ios and md */
```

Comment:
The reason I chose to specify that it updates the `default theme` and not the `light theme` or `light mode` in all of these examples is because the variables assigned to the `:root` selector will apply to **all** themes unless they are overridden specifically for that theme. This is also why I say `when dark mode is enabled` below, to make it clear that those variables will only apply in that case. If I can make this any more clear let me know.

Reply:
So, I believe we should include examples demonstrating how to override the dark theme for a few reasons:

1. This guides developers on the appropriate selectors to use for overriding the existing dark theme, based on the specific dark theme file being used. For example, `dark.always.css` requires overrides in `:root` while `dark.system.css` requires overrides in `@media (prefers-color-scheme: dark) { :root {} }`. This is not covered by the [Colors documentation](https://ionicframework.com/docs/theming/colors#modifying-colors) or the [CSS Variables documentation](https://ionicframework.com/docs/theming/css-variables).
2. It provides an example of overriding theme variables based on the mode. For instance, in `dark.always.css`, `--ion-toolbar-background` is set on `:root.md`, not `:root`. Attempting to override it on `:root` will not work for `md` mode. This aligns with what I mentioned earlier about [including entire theme files for reference](https://github.com/ionic-team/ionic-docs/pull/3384#discussion_r1456506043).
3. It informs developers which file to use for overriding variables to ensure the variables are applied globally. Setting them in a `<style>` tag in Vue, for example, would not work, saving developers from trial and error. Variables should be set in the `variables.css` file.

If we wanted to emphasize more strongly that these overrides are optional, we can do that. Alternatively, we could consider splitting the demos, but this would require adding three more demos because each file has a different way to override variables.

Reply:
Very good points. I'm good with leaving the files. There's also no need to emphasize since the file starts with a comment stating how they're used for overriding.